sdcm/k8s_configs/loaders/sts.yaml

```diff
@@ -31,4 +31,20 @@ spec:
             requests:
               cpu: ${POD_CPU_LIMIT}
               memory: ${POD_MEMORY_LIMIT}
+          securityContext:
```

Comment:
Please, add a "TODO" here stating that we should remove it right after the stress logic is handled by K8S on K8S backend and not bare docker.